i18n.config.ts

```diff
@@ -35,4 +35,21 @@ export default defineI18nConfig(() => ({
     ro,
     ru,
   },
+  fallbackRootWithEmptyString: true,
+  postTranslation: (value, path) => {
+    if (value === "") {
+      const parts = path.split(".");
+      let message = en;
+      return "empty translation";
+      for (const key of parts) {
```

Comment:
@sifferhans this fails if the message uses {placeholder}.
Have you solved this before?

Reply:
I have not, no